frontend/components/Auction.vue

```diff
@@ -233,6 +233,14 @@
                 </div>
             </TextBlock>
         </div>
+        <div v-if="takeEvents" class="mt-5">
+            <p class="mb-2">The auction was taken via {{ takeEvents.length }} transactions.</p>
```

Comment:
```suggestion
            <p class="mb-2">The auction was taken via {{ takeEvents.length }} transactions:</p>
```

Please also handle `s` in the `transactions` in case `length === 1`

Reply:
Sure!